workflow/applications.py

```diff
@@ -362,7 +363,7 @@ def _get_cycled_task_names(self):
 
         if self.do_jediocnvar:
             gdas_gfs_common_tasks_before_fcst += ['ocnanalprep', 'ocnanalbmat', 'ocnanalrun',
-                                                  'ocnanalchkpt', 'ocnanalpost']
+                                                  'ocnanalchkpt', 'ocnanalpost', 'ocnanalvrfy']
```

Comment:
Just curious q. here;
is `ocnanalvrfy` REQUIRED to be run before forecast?  As in, does it produce files that the model needs?  Or is this more of a check to ensure that the ocean initial conditions are sane?

Reply:
@aerorahul , the later, nothing should depend on that task.

Reply:
Thank you.  We will keep in mind when we streamline these ocean jobs and be mindful of not delaying the forecast job if this adds time to trigger.